jekyll/_cci2/artifacts.md

```diff
@@ -124,28 +124,50 @@ When CircleCI runs a build, a link to the core dump file appears under the Artif
 
 ## Downloading All Artifacts for a {% comment %} TODO: Job {% endcomment %}Build on CircleCI  
 
-Use the following procedure to download your artifacts with `curl`.
+To download your artifacts with `curl`,
+follow these steps.
 
-1. Create an API token in the application by going to [User Settings > Personal API Tokens](https://circleci.com/account/api).
+1. On the [Personal API Tokens](https://circleci.com/account/api) settings page,
+click **Create New Token**.
 
-2. Click Create New Token, add a name in the dialog box and click Add API Token. Name tokens to help you remember where they are used.
+2. In the **Token name** field,
+add a name for this token
+to remind you how and where it is used.
 
-3. Copy the token string that appears in the table.
+3. To create the API token,
+click the **Add API Token** button.
 
-4. CD to a directory where you would like the artifacts files to be downloaded and run the following command, copying in the token from Step 3:
+4. Copy the token that appears in the field.
 
-```
+5. In a Terminal window,
+`cd` to a directory
+where you want the artifact files.
+
+6. Run the commands below,
+replacing all variables starting with a `:` with real values for your project.
+`:your_token` should be replaced with the token you copied earlier.
+`:vcs-type` is dependent on your version control system (either `github` or `bitbucket`).
+
+```bash
 export CIRCLE_TOKEN=':your_token'
 
 curl https://circleci.com/api/v1.1/project/:vcs-type/:username/:project/:build_num/artifacts?circle-token=$CIRCLE_TOKEN | grep -o 'https://[^"]*' > artifacts.txt
 
 <artifacts.txt xargs -P4 -I % wget %?circle-token=$CIRCLE_TOKEN
 ```
 
-Note 1: Replace all the variables above that start with a *:* (colon) with real values for your project (don't include the colon).
-
-Note 2: `:vcs-type` will be `github` or `bitbucket`.
-
-Note 3: In the example, the `xargs` command runs four processes to download files in parallel. Adjust this value to your needs.
-
-Explanation: The line beginning with `curl` fetches all the artifacts details for a {% comment %} TODO: Job {% endcomment %}build and pipes it through the `grep` command to extract just the URLs. The results are saved to the `artifacts.txt` file. Then, `xargs` reads the file and downloads each artifact to the current directory.  
+The `curl` command fetches all artifact details for a {% comment %} TODO: Job {% endcomment %} build
+and pipes it through the `grep` to extract the URLs.
+The results are saved to the `artifacts.txt` file.
+Finally, `xargs` reads the file
+and downloads each artifact to the current directory.
+
+In this example, `xargs` runs 4 processes
```

Comment:
4 -> four

Reply:
Is policy to write out all numbers? Or just numbers lower than 10?

I usually use digits just because that's what's being typed in the command, but I can see the argument for spelling it out, too.